src/tools/miri/src/shims/unix/foreign_items.rs

```diff
@@ -1240,26 +1259,21 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
             _ => {
                 let target_os = &*this.tcx.sess.target.os;
                 return match target_os {
-                    "android" =>
-                        android::EvalContextExt::emulate_foreign_item_inner(
-                            this, link_name, abi, args, dest,
-                        ),
-                    "freebsd" =>
-                        freebsd::EvalContextExt::emulate_foreign_item_inner(
-                            this, link_name, abi, args, dest,
-                        ),
-                    "linux" =>
-                        linux::EvalContextExt::emulate_foreign_item_inner(
-                            this, link_name, abi, args, dest,
-                        ),
-                    "macos" =>
-                        macos::EvalContextExt::emulate_foreign_item_inner(
-                            this, link_name, abi, args, dest,
-                        ),
-                    "solaris" | "illumos" =>
-                        solarish::EvalContextExt::emulate_foreign_item_inner(
-                            this, link_name, abi, args, dest,
-                        ),
+                    "android" => android::EvalContextExt::emulate_foreign_item_inner(
```

Comment:
This looks like you ran rustfmt on this file with the wrong settings, please undo that.